tests/test__refresh_worker.py

```diff
@@ -150,7 +150,10 @@ def test_refresh_dead_worker():
 
 def test_pickle():
     w = _refresh_worker.RefreshThreadManager()
+    # For some reason isinstance cannot interpret threading.Lock as a type.
```

Comment:
The reason is that threading.Lock is a function, returning instances of _thread.lock (internal class)